### DISCUSSION TITLE
这个可以跟后端配合做成动态博客吗

### DISCUSSION THREAD

**Discussion by ixcoret** [vuepress-theme-hope/vuepress-theme-hope#3244] [Q&A] [answered]
我是一名Java后端开发，想自己写一个个人博客系统，一直没找到喜欢的前端页面模板。挺喜欢这个UI的，只会一点点前端，想问下这个项目可以跟后端配合做成动态博客吗

**Comment by Mister-Hope** [comment, marked as answer]
这是一个静态博客系统，你可能可以借鉴一部分组件，但是直接复用可能很难。
